Files/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -495,7 +495,7 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
                 {
                     Text = "UnpinItemFromStart2".GetLocalized(),
```

Comment:
```suggestion
                    Text = "UnpinItemFromStart/Text".GetLocalized(),
```

These strings were renamed in `main`.